README.md

```diff
@@ -1,2 +1,33 @@
-# Darko-Online-Class
-This is a microverse first module ending project.
+# Darko Online Class
+This is the website of Darko Music Corporation on which students who are interested in music may use to apply for Darko Music studies. It also provides brief history of Darko Music.
+
+## Built with
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+[Click here](https://christianib003.github.io/Darko-Online-Class/) to view the live demo of the website.
+
+## Get started
+
+- git clone https://github.com/Christianib003/Darko-Online-Class.git
+- cd Darko Online Class
+
+## Authors
+**Author 1**
+- Github: [Christianib003](https://github.com/Christianib003)
+- LinkedIn: [@Christian Iradukunda Byiringiro](https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226)
+
+## Credits
+
+- In this project, I followed the template of [creative commons](https://creativecommons.org/) to make my website. Check them out for more templates.
```

Comment:
- [ ] Good job on mentioning Creative commons but please add appropriate credits to the author of the original design [Cindy Shin](https://www.behance.net/adagio07) as required by the [Creative Commons License](https://creativecommons.org/licenses/). Keep shining 💪